notification/docs/HowToRun.md

```diff
@@ -16,44 +16,127 @@
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-## Environment variables
+## Environment variable
 
-Following environment variables must be provided in order to run the notification module.
+Notification module requires 1 environment variable to start. This environment variable name
+is `ADYEN_INTEGRATION_CONFIG` and it must contain settings as attributes in a JSON structure.
```

Comment:
It's worth to mention that in this module it is not required to have adyen merchant account credentials.

Reply:
It's not written in the doc that you need it. So adding such thing makes it more confusing like why suddenly the doc is mentioning adyen merchant account credentials.